src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -737,6 +772,34 @@ public static RunConfiguration FromXml(XmlReader reader)
                         runConfiguration.TargetPlatform = archType;
                         break;
 
+                    case nameof(DefaultPlatform):
+                        XmlRunSettingsUtilities.ThrowOnHasAttributes(reader);
+                        Architecture defaultArchType;
+                        string defaultPlatformValue = reader.ReadElementContentAsString();
+                        try
+                        {
+                            defaultArchType = (Architecture)Enum.Parse(typeof(Architecture), defaultPlatformValue, true);
```

Comment:
What about using `Enum.TryParse` and avoid the catch?

Reply:
Again, same code as above. We are close to final, I am using what was proven to work. We can redesign it, there is experience for it, but just not now :)